homeassistant/helpers/entity.py

```diff
@@ -253,8 +256,55 @@ def async_update_ha_state(self, force_refresh=False):
                             'https://goo.gl/Nvioub', self.entity_id,
                             end - start)
 
+        def get_overrides(entity_id: str) -> Dict:
+            """Return a dictionary of overrides related to entity_id.
+
+            Whole-domain overrides are of lowest priorities,
+            then glob on entity ID, and finally exact entity_id
+            matches are of highest priority.
+
+            The lookups are cached in a global variable.
+            """
+            result = _OVERWRITE_CACHE.get(self.entity_id)
+            if result is not None:
+                return result
+            domain_result = {}  # type: Dict[str, Any]
+            glob_result = {}  # type: Dict[str, Any]
+            exact_result = {}  # type: Dict[str, Any]
+            domain = split_entity_id(entity_id)[0]
+
+            def clean_entry(entry: Dict) -> None:
+                """Clean up entity-matching keys."""
+                entry.pop('entity_id', None)
+                entry.pop('entity_id_glob', None)
+
+            for rule in _OVERWRITE:
+                if 'entity_id' in rule:
+                    entities = [
```

Comment:
This would not be necessary if you use `cv.entity_ids` in the CUSTOMIZE validator.